ooniprobe/Utility/TestUtility.mm

```diff
@@ -154,11 +156,28 @@ + (void)deleteUploadedJsonsWithMeasurementRemover:(void (^)(Measurement *))remov
     }
 }
 
-//TODO where and when to call it?
 + (void)deleteUploadedJsons{
     [self deleteUploadedJsonsWithMeasurementRemover:^(Measurement *measurement) {
         [TestUtility removeFile:[measurement getReportFile]];
     }];
+    [[NSUserDefaults standardUserDefaults] setObject:[NSDate date] forKey:delete_json_key];
+    [[NSUserDefaults standardUserDefaults] synchronize];
+}
+
++ (BOOL)canCallDeleteJson{
+    NSDate *lastCalled =  (NSDate *)[[NSUserDefaults standardUserDefaults] objectForKey:delete_json_key];
+    
+    if (lastCalled == nil){
+        return YES;
+    }
+    else {
+        NSTimeInterval timeOfNoMagic = [[NSDate date] timeIntervalSinceDate:lastCalled];
```

Comment:
Why is the variable called like this? I don't get it

Reply:
Sorry SO copypasta mistake :D